components/Dialogs/src/MDCAlertController.m

```diff
@@ -465,6 +465,20 @@ - (void)viewWillTransitionToSize:(CGSize)size
                       completion:nil];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+
+  if (self.alertView.contentScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.contentScrollView.frame)) {
+    [self.alertView.contentScrollView flashScrollIndicators];
+  }
+
+  if (self.alertView.actionsScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.actionsScrollView.frame)) {
```

Comment:
Should these be the `bounds` rather than the `frame`?

Reply:
Yep they should be, assuming we need these checks (see my previous comment).